pre-check/v1.x/check.sh

```diff
@@ -556,6 +556,30 @@ check_certs()
     echo -e "\n==============================\n"
 }
 
+check_backup_target()
+{
+    local failed="false"
+    log_info "Starting Backup Target check..."
+
+    version=$(kubectl get settings.harvesterhci.io server-version -o json | jq -r '.value')
+
+    if [ "$version" != "v1.4.1" ]; then
```

Comment:
Thanks, recommend also checking 1.4.2, users might not notice the issue.

Reply:
Updated it. Thanks.